README.md

```diff
@@ -432,22 +432,22 @@ eosc get table eosforumrcpp eosforumrcpp proposal
 ##### Example (get all proposals for a given proposer):
 
 **Caveats** Right now, `eosc` does not support searching giving only a direct key. Instead, it really requires
-a lower and upper bound. The upper bound being exclusive, to correctly get the upper bound, take the name
-is change the last character to the next one in the EOS' name alphabet (order is `a-z0-9.`).
+a lower and upper bound. The upper bound being exclusive, to correctly get the upper bound, take the account name
+and change the last character to the next one in the EOS name alphabet (order is `a-z0-5.`).
 
-So, looking for all proposals proposed by `joshkaufman`, the lower bound key would be `joshkaufman` and
-the upper bound key would be `joshkaufmao` (last character `n` bumped to next one `o`).
+So, looking for all proposals proposed by `testusertest`, the lower bound key would be `testusertest` and
```

Comment:
You don't want to be part of history :P You had existing proposals but thinking about it, they might not exist forever, so I think it's OK.